examples/expression-oriented-style/expression_oriented_style.md

```diff
@@ -0,0 +1,7 @@
+Ballerina's general philosophy is to make the programming syntax familiar to those familiar with the C-family of languages. It has a distinction between statements and expressions, just like in C. This is the imperative style of programming which is considered less intuitive. That's why there is a shift towards a functional style of programming that is more expression-oriented.
```

Comment:
Too much info for a BBE, which is supposed to serve as a quick reference IMO. Can we align it more with the slides?